<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>上传文件</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/新前缀/165.png</normaloff>:/新前缀/165.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	
background-color: rgb(207, 224, 247);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(207, 224, 247), stop:1 rgba(252, 252, 252, 255));

}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	background-color: rgb(252, 252, 252);
}</string>
      </property>
      <property name="text">
       <string>显示图片</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QListView" name="listView">
      <property name="styleSheet">
       <string notr="true">QListView{
	
	font: 11pt &quot;微软雅黑&quot;;
	background-color:  rgb(252, 252, 252);
border-radius: 10px;
border:2px groove gray;
border-style: outset;
	color: rgb(163, 202, 235);
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>百度综艺简体</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton

{
	font: 9pt &quot;百度综艺简体&quot;;
		background-color: rgb(66, 93, 138);
border-radius: 10px;
border:2px groove gray;
border-style: outset;
	color: rgb(252, 252, 252);

}</string>
        </property>
        <property name="text">
         <string>上传文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <family>百度综艺简体</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton

{
	font: 9pt &quot;百度综艺简体&quot;;
		background-color: rgb(66, 93, 138);
border-radius: 10px;
border:2px groove gray;
border-style: outset;
	color: rgb(252, 252, 252);

}</string>
        </property>
        <property name="text">
         <string>开始检测</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
